common/src/Microsoft.Azure.IIoT.Core/src/TSI/Runtime/TsiConfig.cs

```diff
@@ -0,0 +1,28 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Core.TSI.Runtime {
+    using Microsoft.Azure.IIoT.Utils;
+    using Microsoft.Extensions.Configuration;
+    using System;
+
+    /// <summary>
+    /// TSI configuration
+    /// </summary>
+    public class TsiConfig : ConfigBase, ITsiConfig {
+
+        /// <summary>DataAccessFQDN</summary>
+        public string DataAccessFQDN => GetStringOrDefault(PcsVariable.PCS_TSI_URL,
```

Comment:
We usually have a non-environment variable configuration as first configuration source. For example, for `ADLSg2HostName` parameter in `CdmClientConfig.cs` we first try to get value of `Cdm:ADLSg2HostName` before falling back to `PcsVariable.PCS_ADLSG2_ACCOUNT` or others. Can you please add similar configuration here as well ?

Reply:
added